### DISCUSSION TITLE
Are nixpkgs-specific lints in scope for this project?

### DISCUSSION THREAD

**Discussion by viraptor** [oppiliappan/statix#46] [Q&A]
I was planning to write a few lints which are more nixpkgs conventions rather than nix-language lints. For example flagging ordering in some specific files, or redundant platform specifiers for some builders. Are those in scope of statix, or would you rather see them as a separate project? cc @nerdypepper

**Comment by mightyiam** [comment]
Did you or anyone ever write such Nixpkgs lints?
